templates/vc10.mpd

```diff
@@ -799,6 +799,9 @@
 <%endif%>
 <%if(header_files)%>
   <ItemGroup>
+  <%if(pch_header)%> 
```

Comment:
When header_files is empty and pch_header is set it will not be generated?

Reply:
yes,it work well under vs2008 but failed under vs2015